nrt/data/IEEE14/IEEE14_SyntaxExamples/IEEE14_DumpFinalValues/reference/outputs/initValues/globalInit/dumpInitValues-_LOAD___3_EC.txt

```diff
@@ -0,0 +1,31 @@
+ ====== INIT VARIABLES VALUES ======
+load_PRefPu                                       : y =       0.942000 yp =       0.000000
+load_QRefPu                                       : y =       0.190000 yp =       0.000000
+load_deltaP                                       : y =       0.000000 yp =       0.000000
```

Comment:
Why adding those files as reference?

Reply:
I'll remove them but I thought it could be useful.

Reply:
I'll delete nrt/data/IEEE14/IEEE14_SyntaxExamples/IEEE14_DumpFinalValues/outputs/initValues/globalInit and put nrt/data/IEEE14/IEEE14_SyntaxExamples/IEEE14_DumpFinalValues/outputs/initValues/localInit instead